bee-common/bee-common/CHANGELOG.md

```diff
@@ -15,6 +15,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Removed
 
+- Removed `event::Bus` in favour of its inclusion in `bee-runtime`
+
```

Comment:
Include the complete removal of `shutdown`;
Include the removal of `shutdow_stream`, and `worker` moduls for the same reason as the `event` module;

Reply:
Will do